webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -177,56 +198,83 @@ const TrialsComparisonTable: React.FC<TableProps> = (
                 closable
                 key={trial}
                 onClose={() => handleTrialUnselect(trial)}><p>Trial {trial}</p></Tag>)}</div>
-          <div className={css.row}>
-            <h3>State</h3>
-            {trials.map(trial =>
-              <div className={css.centerVertically} key={trial}>
-                <Badge state={trialsDetails[trial].state} type={BadgeType.State} />
-              </div>)}
-          </div>
-          <div className={css.row}>
-            <h3>Training Time</h3>
-            {trials.map(trial =>
-              <p key={trial}>
-                {shortEnglishHumannizer(durations[trial]?.train)}
-              </p>)}
-          </div>
-          <div className={css.row}>
-            <h3>Batches Processed</h3>
-            {trials.map(trial => <p key={trial}>{trialsDetails[trial].totalBatchesProcessed}</p>)}
+          <ul>
+            <li><ul className={css.row}>
+              <h3>State</h3>
+              {trials.map(trial =>
+                <li className={css.centerVertically} key={trial}>
+                  <Badge state={trialsDetails[trial].state} type={BadgeType.State} />
+                </li>)}
+            </ul></li>
+            <li><ul className={css.row}>
+              <h3>Training Time</h3>
+              {trials.map(trial =>
+                <li key={trial}>
+                  {shortEnglishHumannizer(durations[trial]?.train)}
+                </li>)}
+            </ul></li>
+            <li><ul className={css.row}>
+              <h3>Batches Processed</h3>
+              {trials.map(trial =>
+                <li key={trial}>{trialsDetails[trial].totalBatchesProcessed}</li>)}
+            </ul></li>
+            <li><ul className={css.row}>
+              <h3>Total Checkpoint Size</h3>
+              {trials.map(trial => <li key={trial}>{totalCheckpointsSizes[trial]}</li>)}
+            </ul></li>
+          </ul>
+          <div className={css.headerRow}><h2>Metrics</h2>
+            <SelectFilter
+              disableTags
```

Comment:
**chore:** add `allowArrow` prop for `SelectFilter`. But when I enable this I noticed the arrow sits very low. Don't worry about the style, I'll have a separate PR to fix the arrow if you want to just enable it.

Reply:
Ok cool, I tried out the arrow before but turned it off because the positioning was weird.